common/flavorization/wc_title_holders.txt

```diff
@@ -162,149 +132,2398 @@ warlord_female = {
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	culture_groups = { dragon_group }
+}
+lady_female_dragon_king {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	culture_groups = { dragon_group }
+}
+dragonlord_male_dragon_emperor {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	culture_groups = { dragon_group }
+}
+dragonlady_female_dragon_emperor {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	culture_groups = { dragon_group }
+}
+
+### Red Dragons
+dragonprince_male_red_dragon_king {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonprincess_female_red_dragon_king {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonking_male_red_dragon_emperor {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+dragonqueen_female_red_dragon_emperor {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { red_dragon }
+}
+
+### Mantids ###
+emperor_male_mantid {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { mantid }
+}
+empress_female_mantid {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	top_liege = no
+	cultures = { mantid }
+}
+
+imperial_vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = empire
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+imperial_vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = empire
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+grand_vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = kingdom
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+grand_vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = kingdom
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+vizier_male_mantid = {
+	type = character
+	gender = male
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
```

Comment:
Councilor titles should be moved from here to another file.

Reply:
done in 2f11fc97d